x-pack/plugins/features/server/oss_features.ts

```diff
@@ -514,7 +514,7 @@ const reportingFeatures: {
               }
             ),
             includeIn: 'all',
-            minimumLicense: 'platinum',
```

Comment:
This sort of change is also needed on `x-pack/plugins/canvas/server/feature.ts`, line 68